addons/mail/views/mail_template_views.xml

```diff
@@ -23,71 +23,68 @@
                                     groups="base.group_system"
                                     class="oe_stat_button" icon="fa-minus"
                                     attrs="{'invisible':[('ref_ir_act_window','=',False)]}"
-                                    help="Remove the contextual action to use this template on related documents" widget="statinfo">
+                                    help="Remove the contextual action to use this template on related documents"
+                                    widget="statinfo">
                                 <div class="o_field_widget o_stat_info">
                                     <span class="o_stat_text">Remove</span>
                                     <span class="o_stat_text">Context Action</span>
                                 </div>
                             </button>
-                            <button class="oe_stat_button" name="%(mail_template_preview_action)d" icon="fa-search-plus" string="Preview"
+                            <button class="oe_stat_button" name="%(mail_template_preview_action)d"
+                                    icon="fa-search-plus" string="Preview"
                                     type="action" target="new"/>
                         </div>
                         <div class="oe_title">
                             <label for="name"/>
                             <h1><field name="name" required="1" placeholder='e.g. "Welcome email"'/></h1>
                             <group>
-                                <field name="model_id" placeholder="e.g. Contact" required="1" options="{'no_create': True}"/>
                                 <field name="model" invisible="1"/>
                             </group>
                         </div>
                         <notebook>
                             <page string="Content" name="content">
                                 <div class="oe_title"><label for="subject"/></div>
                                 <div class="oe_title">
-                                    <h2 style="display: inline-block;"><field name="subject" placeholder='e.g. "Welcome to MyCompany" or "Nice to meet you, ${object.name}"'/></h2>
+                                    <field name="subject"
+                                            placeholder='e.g. "Welcome to MyCompany" or "Nice to meet you, ${object.name}"'/>
```

Comment:
While we are at it, note that jinja syntax is not supporter since v15 ^^ we should probably improve the placeholder, like ``e.g. "Welcome to MyCompany" or "Nice to meet you, {{ object.name }}"``